querybook/webapp/components/DataDocLeftSidebar/DataDocLeftSidebar.scss

```diff
@@ -26,9 +26,7 @@
         width: 100%;
         height: calc(100vh - 158px);
         max-height: 1200px;
-    }
 
-    .sidebar-content-table {
```

Comment:
why remove this?

Reply:
Seemed redundant since the styles above also applied to the same class. 

Can revert.

Reply:
Looks like the above css applies to both table and content, but this `.sidebar-content-table` only applies to the table. Let's revert this change in case, unless you've fully tested it and it looks good.